### ISSUE TITLE
Memcached, php 7+ support? Fast?

### ISSUE THREAD

**Issue by 1stmetro** [commentics/commentics#2]
Not used this system for a while it used to load a lot if files, images etc.

Has it been streamlined so it loads faster?

Will put the comments in memcached and update when a new comment has been posted, has this feature been added yet?

**Comment by commentics** [comment]
Yes the newest version of Commentics uses sprites, CDNs, ajax for a faster UX, supports PHP 7 and contains many database query optimizations.

I've worked a lot with Memcached recently so I know how beneficial it is, however I have two concerns with its suitability for Commentics. Firstly the update would not only be when posting a comment but also when liking/disliking which are more frequent and consequently lessens the advantage of caching. Secondly is how Memcached isn't typically installed on shared hosting environments and I'd like for everyone to use it. It would have to be an option next to others like file-based.

**Comment by 1stmetro** [comment]
I have a million pages so I have to use memcache to minimise the hammering the server can get.

Did you leave the memcache code in or I would have to add it?

**Comment by commentics** [comment]
It wasn't Commentics that I implemented it on.. so you would have to add it.

**Comment by shoaibshafiq7** [comment]
I have installed the commentics but http://localhost/commentics/comment/frontend/ is not working..
any solution to solve?
